backend/src/layer/data/flickr_api/flickr_api_handler.rs

```diff
@@ -85,6 +86,7 @@ impl ImageHoster for FlickrApiHandler {
     /// # Return
     /// If the image exists, the [`ImageMetaData`] struct will be returned.
     async fn validate_url(&self, url: &str) -> Result<ImageMetaData> {
+        trace!("FlickrApi: Started url validation with '{url}'");
```

Comment:
too many log messages!
i would remove them here and keep those when requestig external sources